xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -3299,6 +3309,15 @@ void CVideoPlayer::GetGeneralInfo(std::string& strGeneralInfo)
         strBuf += StringUtils::Format(" {} msec", DVD_TIME_TO_MSEC(m_State.cache_delay));
     }
 
+    XFILE::SCacheStatus status;
+    if (m_pInputStream->GetCacheStatus(&status))
+    {
+      strBuf +=
+          StringUtils::Format("\nCurrent rate: {}/s", StringUtils::SizeToString(status.currate));
+      strBuf += StringUtils::Format(", low: {}/s", StringUtils::SizeToString(status.lowrate));
+      strBuf += StringUtils::Format(", max: {}/s", StringUtils::SizeToString(status.maxrate));
```

Comment:
`StringUtils::SizeToString` is for size units (MB megabytes) and bitrate units are probably Mbit/s

For it to make sense, the sum of the video bitrate + audio bitrate should be similar to the cache bitrate (or file reading rate if FileCache is not used).

Note that video and audio bitrates are displayed in the the first lines of debug OSD (aq, vq) and are expressed in Mb/s and Kb/s (Mbits/s and Kbits/s).

Reply:
Concerning code has been removed.